fluentd/Dockerfile.centos7

```diff
@@ -7,7 +7,10 @@ ENV DATA_VERSION=1.6.0 \
     GEM_HOME=/opt/app-root/src \
     HOME=/opt/app-root/src \
     PATH=/opt/app-root/src/bin:/opt/app-root/bin:$PATH \
-    RUBY_VERSION=2.0
+    RUBY_VERSION=2.0 \
+    LOGGING_FILE_PATH=/opt/app-root/src/fluentd.log \
```

Comment:
This should default to the same volume used by fluent to store its buffers.

Reply:
Is that `/var/log/fluentd/` or `/var/log/` or `/var/lib/fluentd/`?

Reply:
So maybe not [1] since this is in `/var/lib`.  We should defer then to using conventions: `/var/log/fluentd`

[1] https://github.com/openshift/origin-aggregated-logging/blob/master/fluentd/run.sh#L207